shell.nix

```diff
@@ -1,9 +1,5 @@
-{ crossSystem ? null
-, system ? builtins.currentSystem
-, config ? { allowUnfreePredicate = (import ./nix/lib/unfree.nix).unfreePredicate; }
-, sourcesOverride ? { }
-, packages ? import ./. { inherit crossSystem config sourcesOverride enableHaskellProfiling; }
-, enableHaskellProfiling ? false
```

Comment:
I care about this one! `nix-shell --arg enableHaskellProfiling true` is how you get a profiling shell.

Reply:
oops, sorry. Didn't remove this on purpose. Will bring it back..

Reply:
Fixed in https://github.com/input-output-hk/plutus/pull/3221/commits/08a2cfeb4336619520b92769f912a41bac776376